### ISSUE TITLE
Escape special character in PDO

### ISSUE THREAD

**Issue by FWAJL** [FWAJL/FieldWorkAssistantMVC#425]
escape single quote

**Comment by FWAJL** [comment]
@bsaiken : this is more tricky than I thought. We need to use the PDO class of PHP to build clean sql queries. 
The single quote is a character reserved for the sql syntax and that's why it fails when add one in a data string.

**Comment by bsaiken** [comment]
@WebDevJL 
This is a very low priority item, but since you looked into it anyway, it is clear that your proposed solution is too complicated and not the way to go.  This can be solved by adding front-end data validation, which will screen out potential offending strings.  
Close this Issue and create a new issue for data validation.  Include this specific issue in the comments.  The issue can be low priority for now.

**Comment by FWAJL** [comment]
@bsaiken 

We can leave this opened to remember it needs to be done. We'll link it the front end validation task.

**Comment by FWAJL** [comment]
Temporary solution awaiting to find resources to do: #426

**Comment by bsaiken** [comment]
I don't understand why, since you have created Issue #446, this Issue stays open.  Every open Issue is one more thing I need to look at every week.

**Comment by FWAJL** [comment]
@bsaiken 

Because I want to keep in mind that this the best solution even if it is complex to implement. We can't just get rid of a problem by ignoring it. See my email about moving the issues you don't want to see elsewhere.

Thank you.

**Comment by FWAJL** [comment]
@bsaiken 

We can close it only once we create the issue in [the backlog](https://github.com/FWM-Team/FWM-Backlog-Issues/issues). This one is.
